### PR TITLE
46551 update store types

### PR DESCRIPTION
**Pull request by rcpokorny**
Updated the integration-manifest json file so the settings would match the values in the Read Me documentation.

### CODE REVIEW

**Comment by RexWheeler** [comment]
@rcpokorny Hey the cert store types stuff in the manifest still doesn't match the readme... The question marks were removed on some of the display names, and other displayname where changed as well. Please verify that all the parameters and entry parameters match what is in the readme.

File: integration-manifest.json [hunk 1]
```diff
@@ -285,20 +306,24 @@
...
             }
           ],
           "PasswordOptions": {
             "EntrySupported": false,
             "StoreRequired": false,
             "Style": "Default"
           },
-          "StorePathValue": "[\"My\",\"WebHosting\"]",
+          "StorePathType": "",
+          "StorePathValue": "",
```
Comment (joynerar) [inline integration-manifest.json:322]: Not sure if you're still working on this @rcpokorny, but for IISU we're going to want to restrict the store paths to My and WebHosting via a multiple-choice type.

**Comment by rcpokorny** [comment]
Thanks Adam - I am still testing the KFUtil application.  It appears that some properties are being missed.  I will notify Sean that the property has been removed from the json.

**Comment by bhillkeyfactor** [comment]
@RexWheeler @rcpokorny @fiddlermikey - What is the status of this PR? I have another coming into the dev branch shortly for SQL Bindings.

**Comment by rcpokorny** [comment]
This PR has all the updated changes to the integration manifest to match that of the updated read me.
I was working with @Sean ***@***.***> on the KFUtil, making sure the settings were being created correctly.  Sean was having problems with KFUtil and said he needed to do some work.  My guess is this fell through the cracks.

I copied Sean on this email, let's see if we can get an update from him.  If KFUtil is secondary, we can merge this PR and move forward.

**Pull request merged**